api/src/main/java/com/codeforcommunity/dto/user/UserListResponse.java

```diff
@@ -0,0 +1,35 @@
+package com.codeforcommunity.dto.user;
+
+
+import java.util.List;
+
+public class UserListResponse {
+
+  private int count;
+  private List<UserDataResponse> users;
```

Comment:
Let's not include count. We never use it on the frontend. It takes constant time to get the length of an array.